module/spl/spl-proc.c

```diff
@@ -476,6 +695,20 @@ spl_proc_init(void)
 		goto out;
 	}
 
+	proc_spl_taskq_all = proc_create_data("taskq-all", 0444,
+		proc_spl, &proc_taskq_all_operations, NULL);
+	if (proc_spl_taskq_all == NULL) {
+		rc = -EUNATCH;
```

Comment:
The use of EUNATCH is a really strange choice. I know you just copy the above ones. But perhaps we should change them all to ENOMEM.